util/buildRelease/smokeTest

```diff
@@ -9,6 +9,68 @@ set -e
 
 CWD=$(cd $(dirname $0) ; pwd)
 
+MASON=0
+DOCS=0
+HELLO=0
```

Comment:
I don't love `hello` as the target name, but `check` isn't very helpful either. Maybe `makeCheck` is a little better? I guess the part I don't love is hardcoding that `make check` runs one of the hellos. Not a big deal, I'm fine leaving this as-is if you like it.

Reply:
Yeah. Initially I called it `check` but I felt that was useless because all of the things are "checks". Maybe we should just call it `chpl` on the theory that `mason` would build mason and check mason, and `docs` builds `chpldoc` and checks it by running the docs build?

Reply:
Yeah, I like `chpl`